services/ui-src/src/components/forms/Form.tsx

```diff
@@ -24,6 +25,7 @@ export const Form = ({
   ...props
 }: Props) => {
   const { fields, options } = formJson;
+  const selectedReport = localStorage.getItem("selectedReport");
```

Comment:
I think this is gonna be a problem for non-report forms, like the admin banner page.

Reply:
@gmrabian I got a solution added in for that, let me know what you think